packages/wrangler/src/r2/sippy.ts

```diff
@@ -103,12 +121,24 @@ export async function EnableHandler(
 				args.serviceAccountKeyFile = await prompt(
 					"Enter the path to your Google Cloud service account key JSON file:"
 				);
+				if (!args.serviceAccountKeyFile) {
+					throw new UserError(
+						"Must specify the path to a service account key JSON file."
+					);
+				}
 			}
 		}
```

Comment:
We should probably have an else clause here checking that a valid provider has been specified, instead of at the prompting level, then LGTM :)